src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import combineReducers from "redux/src/combineReducers";
+
+import  listBooksReducer from "./listBooksReducer"
+import selectedBookReducer from "./selectedBookReducer";
+export  default  combineReducers({
+    listBooksReducer,
```

Comment:
`listBooksReducer` can be `booksList: listBooksReducer`, just as it is a state object, having the word 'Reducer' inside key is a bit confusing,
This comment is optional, there's no naming convention

Reply:
Right ..